log.py

```diff
@@ -35,17 +33,18 @@ def only_native_strings(s):
             we don't want to try to interpolate an incompatible type; it
             could lead to a UnicodeDecodeError.
             """
-            if isinstance(s, (bytes, unicode)):
-                s = native_string(s)
+            if isinstance(s, bytes):
+                s = s.decode("utf-8")
             return s
+        message = only_native_strings(record.msg)
```

Comment:
This was originally designed for forward-compatibility with Py3. Now that we're moving over it could be simplified.

Reply:
Maybe the name of the function is confusing because of the conversion, but there are occasions where the message or arguments can be a byte string so this is still useful. Maybe changing it to `use_string(...)` makes more sense?